composer.json

```diff
@@ -192,7 +192,6 @@
     "reset:themes" : "rm -rf public/wp-content/themes",
     "reset:plugins" : "rm -rf public/wp-content/plugins",
 
-    "download:wordpress": "wp core download --version=5.0.3 --path=public --force",
```

Comment:
Perhaps we can make this work with the WP version defined in this file. But not sure if that would be useful since nothing seems to depend on this atm.